service/cluster/xtra_db_cluster.go

```diff
@@ -68,22 +69,22 @@ func (s *XtraDBClusterService) ListXtraDBClusters(ctx context.Context, req *cont
 		params := &controllerv1beta1.XtraDBClusterParams{
 			ClusterSize: cluster.Size,
 			Pxc: &controllerv1beta1.XtraDBClusterParams_PXC{
-				DiskSize: cluster.PXC.DiskSize,
+				DiskSize: convertors.StrToBytes(cluster.PXC.DiskSize),
```

Comment:
isn't it better to move calling converters inside k8client?

Reply:
To not leave a chance to use wrong suffixes

Reply:
I've considered such case - but decided to call it here as we convert values in communication between dbaas-controller and pmm-managed but not with Kubernetes API.